pkg/querier/stats/query_metrics.go

```diff
@@ -33,5 +36,10 @@ func NewQueryChunkMetrics(reg prometheus.Registerer) *QueryChunkMetrics {
 			Name:      "distributor_query_ingester_chunks_total",
 			Help:      "Number of chunks received from ingesters at query time, including any chunks discarded as duplicates.",
 		}),
+		QueriesExceededLimits: promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
+			Namespace: "cortex",
+			Name:      "queries_exceeded_limits_total",
```

Comment:
[nit] Should we prefix with "cortex_querier" to keep it consistent with other querier metrics? These limits are enforced only in the querier, right? I don't feel strong about this comment.

```suggestion
			Name:      "querier_queries_exceeded_limits_total",
```

Reply:
Changed in b917f19